frontend/src/lib/Utils.tsx

```diff
@@ -358,3 +359,24 @@ export function buildQuery(queriesMap: { [key: string]: string | number | undefi
   }
   return `?${queryContent}`;
 }
+
+export async function decodeCompressedNodes(compressedNodes: string): Promise<object> {
+  return new Promise<object>(resolve => {
+    const compressedBuffer = Buffer.from(compressedNodes, 'base64');
+    zlib.gunzip(compressedBuffer, { windowBits: 15 }, (error, result: Buffer) => {
```

Comment:
nit: what does the `windowBits: 15` mean?

Can we avoid the magic number?

Reply:
Sorry forgot to remove it.